docs/workflow_solutions/git_collaboration.md

```diff
@@ -94,7 +94,7 @@ The workflow assumes a central repository already exists within an organization
     1. [Clone](git.md#cloning) the downstream fork to the local machine where the programming will happen.
 - Workflow
     1. Decide on a set of changes to make. Good practice is only working on one conceptual unit at a time. One feature, one bug fix, or one documentation page. Prefer fixing bugs before adding features.
-    1. Synchronize your downstream fork with the upstream fork to minimize risk of merge conflicts. [GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) [GitLab](https://docs.gitlab.com/ee/user/project/repository/forking_workflow.html#repository-mirroring)
+    1. Synchronize your downstream fork with the upstream fork to minimize risk of merge conflicts. [GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) [GitLab](https://docs.gitlab.com/ee/user/project/repository/forking_workflow.html)
```

Comment:
Oh good catch. I think we may want this one though: https://docs.gitlab.com/ee/user/project/repository/forking_workflow.html#update-your-fork